admin_view/nuxt-project/pages/stock_items/_id.vue

```
@@ -0,0 +1,959 @@
+      stockerItemName: "",
+      stockerItemNum: null,
+      rentableItems: [],
+      stockerItems: [],
+      stockerPlaces: [],
+      groups: [],
+      assignGroup: "",
+      assignRentalItems: [],
+      itemYear: [],
+      itemFesYear: "",
+      itemStockerPlaceId: "",
+      roomName: null,
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)